src/main/kotlin/nl/avisi/kotlinwebtest/jdbc/Validation.kt

```diff
@@ -0,0 +1,52 @@
+package nl.avisi.kotlinwebtest.jdbc
+
+import nl.avisi.kotlinwebtest.ExecutionContext
+import nl.avisi.kotlinwebtest.Validator
+import nl.avisi.kotlinwebtest.ValidatorResult
+import nl.avisi.kotlinwebtest.expressions.Expression
+import nl.avisi.kotlinwebtest.expressions.ExpressionEvaluator
+
+val noResults: String = "There are no result found with the given query"
+val columnDoesntExists: String = "The given column name doesnt exists"
+
+class QueryValidator(private val columnName: String, private val position: Int, private var expectedValue: Expression? = null) : Validator<JdbcStepRequest, JdbcStepResponse> {
+
+    override fun validate(executionContext: ExecutionContext, request: JdbcStepRequest, response: JdbcStepResponse): ValidatorResult {
+        if (response.resultSet!!.isEmpty()) return failure(noResults)
+        if (!listContainsColumnName(response.resultSet[position], columnName)) return failure(columnDoesntExists)
+        if (response.resultSet.size <= position) return failure("There are not enough result with given index")
+
+        val resultValue = response.resultSet[position].getValue(columnName)
+        val expectedValue = expectedValue?.let { ExpressionEvaluator(executionContext).evaluate(it) } ?: error("QueryValidator is missing expected value")
+
+        if (resultValue != expectedValue) return failure("no match found for: $columnName")
+        return success()
+    }
+}
+
+class RowCountValidator(var expectedValue: Int) : Validator<JdbcStepRequest, JdbcStepResponse> {
+
+    override fun validate(executionContext: ExecutionContext, request: JdbcStepRequest, response: JdbcStepResponse): ValidatorResult {
+        val amount: Int = response.resultSet!!.size
+        if (amount != expectedValue) return failure("there were $amount row(s) instead of $expectedValue row(s)")
+        return success()
+    }
+}
+
+class BooleanValidator(val columnName: String, val position: Int, var expectedValue: Boolean) : Validator<JdbcStepRequest, JdbcStepResponse> {
+
+    override fun validate(executionContext: ExecutionContext, request: JdbcStepRequest, response: JdbcStepResponse): ValidatorResult {
```

Comment:
I feel there's some duplication with other JDBC validators here. Maybe some abstraction or refactoring is in order?